src/components/NavBar.tsx

```diff
@@ -15,7 +15,7 @@ const NavBar = () => {
   };
 
   return (
-    <nav className="flex justify-between items-center h-16 fixed z-50 w-full p-8 pt-16 md:p-20">
+    <nav className="flex justify-between items-center h-16 fixed z-50 w-full p-8 pt-16 md:p-20 res">
```

Comment:
- try to use responsive breakpoint given in the tailwind, try to avoid writing custom breakpoints